Sources/SwiftKuery/Table.swift

```diff
@@ -138,9 +251,25 @@ open class Table: Buildable {
             onCompletion(.error(QueryError.syntaxError("\(error)")))
         }
     }
-    
-    /// Add a primary key to the table.
-    ///
+    // MARK: Assign Keys
+    /**
+     Function to set multiple `Column` instances, as a composite primary key, in the `Table` instance.
```

Comment:
How about...
"Creates a composite primary key, from multiple `Column` instances, in the `Table` instance.

The columns are validated to ensure that they belong to the table and do not conflict with the definition of a primary key."